packages/mobile/src/type/index.ts

```diff
@@ -7,7 +7,7 @@ export type Restaurant =
   | "한빛식당"
   | "선택안함";
 
-export type FlickingMenu = { id: number; name: string };
+export type Menu = { id: number; name: string };
```

Comment:
이 부분 식당 페이지의 메뉴에 쓰이는 타입이라고 타입명에 명시해도 좋을 것 같습니다!
```suggestion
export type RestaurantMenu = { id: number; name: string };
```

Reply:
원래 Map에서도 MenuList를 사용했어서 나중을 생각하면 그대로 두는게 좋을 것 같습니다!